src/styles/main1.scss

```diff
@@ -0,0 +1,299 @@
+body {
```

Comment:
What's the purpose of having both main.scss and main1.scss? And why do you have style for different BEM block inside of them?